tools/build/build.sh

```diff
@@ -20,21 +20,17 @@ set -e pipefail
 # Display commands
 set -x
 
-retry() {
-  "${@}" || "${@}" || "${@}" || exit $?
-}
-
 cd $(dirname $0)/../..
 BASE_DIR=$PWD
 
 ARTIFACTS_OUT="${BASE_DIR}/artifacts"
 mkdir -p "$ARTIFACTS_OUT"
 
-retry npm install
+npm install
 
 for version in 6.0.0 8.0.0 10.0.0 11.0.0 12.0.0
 do
-  retry ./node_modules/.bin/node-pre-gyp configure rebuild package \
+  ./node_modules/.bin/node-pre-gyp configure rebuild package \
```

Comment:
If all succeed except one version, the current change will retry all versions even those that succeeded. 
If "npm install" happens to be slow, it will be retried as well. Are there any concerns with timeout in such cases?

Reply:
Discussed offline.
This trade off is simpler than retries sprinkled throughout the code.